Modules/Core/Common/include/itkVariableLengthVector.h

```diff
@@ -314,6 +315,17 @@ class ITK_TEMPLATE_EXPORT VariableLengthVector
   /** Typedef used to indicate the number of elements in the vector */
   using ElementIdentifier = unsigned int;
 
+  // Nested types from C++ Standard section "Container requirements" [container.reqmts], and from `std::vector`:
+  using value_type = TValue;
+  using reference = TValue &;
+  using const_reference = const TValue &;
+  using iterator = TValue *;
+  using const_iterator = const TValue *;
+  using difference_type = itk::ptrdiff_t;
```

Comment:
Note to self: `itk::` is not really necessary here, as we are already in the `itk` namespace. That's no show-stopper, though 😃